{{cookiecutter.module_name}}/Makefile

```diff
@@ -100,18 +100,21 @@ json/format: | guard/program/jq
 	$(FIND_JSON) | $(XARGS) bash -c 'echo "$$(jq --indent 4 -S . "{}")" > "{}"'
 	@ echo "[$@]: Successfully formatted JSON files!"
 
-docs/%: README_PARTS := _docs/MAIN.md <(echo) <(terraform-docs markdown table .)
-docs/%: README_FILE ?= README.md
+tfdocs-awk/install: $(BIN_DIR)
+tfdocs-awk/install: ARCHIVE := https://github.com/plus3it/tfdocs-awk/archive/master.tar.gz
+tfdocs-awk/install:
+	$(CURL) $(ARCHIVE) | tar -C $(BIN_DIR) --strip-components=1 --wildcards '*.sh' --wildcards '*.awk' -xzvf -
+	chmod -R 755 $(BIN_DIR)/*
 
-docs/lint: | guard/program/terraform-docs
-	@ echo "[$@]: Linting documentation files.."
-	diff $(README_FILE) <(cat $(README_PARTS))
-	@ echo "[$@]: Documentation files PASSED lint test!"
-
-docs/generate: | guard/program/terraform-docs
+docs/generate: | tfdocs-awk/install guard/program/terraform-docs
 	@ echo "[$@]: Creating documentation files.."
-	cat $(README_PARTS) > $(README_FILE)
-	@ echo "[$@]: Documentation files creation complete!"
+	@ bash autodocs.sh -g
```

Comment:
avoid exiting with false success using `bash -eu -o pipefail ...`

Reply:
updated

Reply:
@lorengordon - doesn't this already happen given that the Makefile defined the shell/shellflags

```
SHELL := bash
.SHELLFLAGS := -eu -o pipefail -c
```

Reply:
The makefile shellflags are set on the shell that make runs the command in. If the command is _itself_ invoking a shell (e.g. `bash ...`), then _maybe_ the shellflags are inherited. Haven't traced out the behavior enough to figure out why sometimes shell flags are inherited and sometimes not, so I generally just make sure to specify them whenever calling a shell.